src/CONST.js

```diff
@@ -1069,264 +1069,1084 @@ const CONST = {
+            samples: 'TKCA 1ZZ',
+        },
+        TD: {},
+        TF: {},
+        TG: {},
+        TH: {
+            regex: /^\d{5}$/,
+            samples: '30706, 18695, 21044, 42496',
+        },
+        TJ: {
+            regex: /^\d{6}$/,
+            samples: '381098, 961344, 519925, 667883',
+        },
+        TK: {},
+        TL: {},
+        TM: {
+            regex: /^\d{6}$/,
+            samples: '544985, 164362, 425224, 374603',
+        },
+        TN: {
+            regex: /^\d{4}$/,
+            samples: '6075, 7340, 2574, 8988',
+        },
+        TO: {},
+        TR: {
+            regex: /^\d{5}$/,
+            samples: '42524, 81057, 50859, 42677',
+        },
+        TT: {
+            regex: /^\d{6}$/,
+            samples: '041238, 033990, 763476, 981118',
+        },
+        TV: {},
+        TW: {
+            regex: /^\d{3}(?:\d{2})?$/,
+            samples: '21577, 76068, 68698, 08912',
+        },
+        TZ: {},
+        UA: {
+            regex: /^\d{5}$/,
+            samples: '10629, 81138, 15668, 30055',
+        },
+        UG: {},
+        UM: {},
+        US: {
+            regex: /^[0-9]{5}(?:[- ][0-9]{4})?$/,
+            samples: '12345, 12345-1234, 12345 1234',
+        },
+        UY: {
+            regex: /^\d{5}$/,
+            samples: '40073, 30136, 06583, 00021',
+        },
+        UZ: {
+            regex: /^\d{6}$/,
+            samples: '205122, 219713, 441699, 287471',
+        },
+        VA: {
+            regex: /^(00120)$/,
+            samples: '00120',
+        },
+        VC: {
+            regex: /^VC\d{4}$/,
+            samples: 'VC0600, VC0176, VC0616, VC4094',
+        },
+        VE: {
+            regex: /^\d{4}$/,
+            samples: '9692, 1953, 6680, 8302',
+        },
+        VG: {
+            regex: /^VG\d{4}$/,
+            samples: 'VG1204, VG7387, VG3431, VG6021',
+        },
+        VI: {
+            regex: /^(008(?:(?:[0-4]\d)|(?:5[01])))(?:[ -](\d{4}))?$/,
+            samples: '00820, 00804 2036, 00825 3344, 00811-5900',
+        },
+        VN: {
+            regex: /^\d{6}$/,
+            samples: '133836, 748243, 894060, 020597',
+        },
+        VU: {},
+        WF: {
+            regex: /^986\d{2}$/,
+            samples: '98692, 98697, 98698, 98671',
+        },
+        WS: {
+            regex: /^WS[1-2]\d{3}$/,
+            samples: 'WS1349, WS2798, WS1751, WS2090',
+        },
+        XK: {
+            regex: /^[1-7]\d{4}$/,
+            samples: '56509, 15863, 46644, 21896',
+        },
+        YE: {},
+        YT: {
+            regex: /^976\d{2}$/,
+            samples: '97698, 97697, 97632, 97609',
+        },
+        ZA: {
+            regex: /^\d{4}$/,
+            samples: '6855, 5179, 6956, 7147',
+        },
+        ZM: {
+            regex: /^\d{5}$/,
+            samples: '77603, 97367, 80454, 94484',
+        },
+        ZW: {},
+    },
+
+    GENERIC_ZIP_CODE_REGEX: /^(?:(?![\s-])[\w -]{0,9}[\w])?$/,
```

Comment:
Here - should we increase this to 15?

```suggestion
    GENERIC_ZIP_CODE_REGEX: /^(?:(?![\s-])[\w -]{0,15}[\w])?$/,
```

Reply:
Actually in slack [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1678738509974729?thread_ts=1678270271.367389&cid=C01GTK53T8Q) we decided 0 - 10 should be fine for now

Reply:
True true but on second thought i think 0-10 characters is fine for now, we can adjust later if needed